docs/migration.md

```
@@ -0,0 +1,187 @@
+## Tested scenarios
+
+Local end-to-end run against `minio/minio:latest` (Docker) and `rclone/rclone:latest`
+(Docker). Source bucket contained:
+
+- 100 small objects (`k-001.txt` … `k-100.txt`, single PUT each)
+- 1 × 50 MiB random object (`big.bin`, transferred as a 7-part multipart upload)
+- 1 object with custom `Content-Type: application/pdf` and a user-metadata header
+
+Sync command (`rclone/rclone:latest`, `list_version = 2` set on the destination remote):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)